python-api/gstswitch/server.py

```diff
@@ -247,14 +355,16 @@ def _start_process(self, cmd):
         :param cmd: The command which needs to be executed
         :returns: process created
         """
-        print('Creating process %s' % (cmd))
+        self.log.info('Starting process %s', cmd)
         try:
             if self.log_to_file:
```

Comment:
This feature was introduced in #193 so I incorporated those changes into this PR, too…